catalogs/qhimm.xml

```diff
@@ -2222,4 +2222,37 @@ Everyone has a Curse Ring Equipped, the challenge is to win before Death Sentanc
     </Mod>
 
   </Mods>
+
+## WereXIII - by Wolfman
+    <Mod>
+      <ID>1iwfYEec-eu6K4hB6zD06dok1jH92lX3n</ID>
+      <Author>Tsunamods</Author>
+      <Link>https://drive.google.com/file/d/1iwfYEec-eu6K4hB6zD06dok1jH92lX3n/view?usp=drive_link</Link>
+      <LatestVersion>
+        <Link></Link>
+        <Version>3.0</Version>
+        <ReleaseDate>2014-06-10</ReleaseDate>
+        <CompatibleGameVersions>All</CompatibleGameVersions>
+        <PreviewImage></PreviewImage>
+        <ReleaseNotes/>
+        <DownloadSize>419</DownloadSize>
```

Comment:
This should be in KB ( just open up an explorer window in Windows and put the same number it gives you in KB )